content/_en/help/verify-your-identity/phone-number-and-phone-plan-in-your-name.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: help
```

Comment:
I see it's the same in other pages as well, but... is there any reason this is in English version and not the others? Is it doing anything?

Reply:
@aduth I have noticed this too. It is something that I have investigated passively in the past. I am leaning to the conclusion that it may not do anything because the layout is based on the kind of page it is, but I cannot say that definitively.

Reply:
I think the variable defaults to `help` anyways, based on this configuration:

https://github.com/18F/identity-site/blob/78e297693b56156e2823022e758029ac7ec30466/_config.yml#L58-L63

So it's redundant, but doesn't harm anything either, which likely explains why we've not seen an issue with differences between language content. In the future, maybe we can clean this up and just remove the `layout: help` from all the help page content frontmatter.